src/components/AppNavigationItem/InputConfirmCancel.vue

```diff
@@ -28,24 +28,44 @@
 ```
 </docs>
 <template>
-	<div class="app-navigation-entry__inline-input-container">
-		<form @submit.prevent="confirm" @keydown.esc.exact.prevent="cancel" @click.stop.prevent>
+	<div class="app-navigation-input-confirm">
+		<form @submit.prevent="confirm"
+			@keydown.esc.exact.prevent="cancel"
+			@click.stop.prevent>
 			<input ref="input"
 				v-model="valueModel"
 				type="text"
-				class="app-navigation-entry__inline-input"
+				class="app-navigation-input-confirm__input"
 				:placeholder="placeholder">
+
 			<button type="submit"
-				class="icon-confirm"
-				@click.stop.prevent="confirm" />
+				class="app-navigation-input-confirm__confirm"
+				:aria-label="t('Confirm changes')"
```

Comment:
Ahh, sorry i didnt realise before: that might produce the same problem on translation extraction as we currently had...
Same below on the other button.

Reply:
Ah, forgot about this! :+1:

Reply:
Done! :heart: